workspaces/multi-source-security-viewer/plugins/multi-source-security-viewer/package.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "@backstage-community/plugin-multi-source-security-viewer",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/backstage/community-plugins",
+    "directory": "workspaces/multi-source-security-viewer/plugins/multi-source-security-viewer"
+  },
+  "backstage": {
+    "role": "frontend-plugin",
+    "pluginId": "multi-source-security-viewer",
+    "pluginPackages": [
+      "@backstage-community/plugin-multi-source-security-viewer",
+      "@backstage-community/plugin-multi-source-security-viewer-common"
+    ]
+  },
+  "sideEffects": [
+    "./**/*.css",
+    "./**/*.css.esm.js"
+  ],
+  "scalprum": {
+    "name": "backstage-community.plugin-multi-source-security-viewer",
+    "exposedModules": {
+      "PluginRoot": "./src/index.ts"
+    }
+  },
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "dependencies": {
+    "@aonic-ui/pipelines": "^3.1.1",
+    "@backstage-community/plugin-github-actions": "^0.7.0",
+    "@backstage-community/plugin-jenkins": "^0.15.0",
+    "@backstage-community/plugin-multi-source-security-viewer-common": "workspace:^",
+    "@backstage/catalog-model": "^1.7.1",
+    "@backstage/core-components": "^0.16.1",
+    "@backstage/core-plugin-api": "^1.10.1",
+    "@backstage/integration-react": "^1.2.3",
+    "@backstage/plugin-catalog-react": "^1.14.2",
+    "@backstage/plugin-permission-react": "^0.4.30",
+    "@backstage/theme": "^0.6.2",
+    "@immobiliarelabs/backstage-plugin-gitlab": "^6.8.0",
+    "@janus-idp/shared-react": "^2.14.0",
+    "@material-ui/core": "^4.9.13",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "^4.0.0-alpha.61",
+    "@mui/icons-material": "5.16.14",
+    "@mui/material": "5.16.14",
+    "@patternfly/patternfly": "^6.0.0",
+    "@patternfly/react-core": "^6.0.0",
+    "@patternfly/react-tokens": "^6.0.0",
+    "@tanstack/react-query": "^5.62.7",
+    "lodash": "^4.17.21"
+  },
+  "peerDependencies": {
+    "react": "^16.13.1 || ^17.0.0 || ^18.0.0",
+    "react-dom": "^16.13.1 || ^17.0.0 || ^18.0.0",
```

Comment:
```suggestion
    "react-dom": "^17.0.0 || ^18.0.0",
```

Reply:
The latest version of Backstage drops support for react 16 so let's not add it in here 👍

Reply:
Removed